pkg/registry/apis/query/query.go

```diff
@@ -369,8 +369,8 @@ func (b *QueryAPIBuilder) handleExpressions(ctx context.Context, req parsedReque
 			if !ok {
 				dr, ok := qdr.Responses[refId]
 				if ok {
-					allowLongFrames := false // TODO -- depends on input type and only if SQL?
-					_, res, err := b.converter.Convert(ctx, req.RefIDTypes[refId], dr.Frames, allowLongFrames)
+					//allowLongFrames := false // TODO -- depends on input type and only if SQL?
```

Comment:
I'm not sure why conversion is happening in api and not inside the SSE service. Will need to look into that, and probably bring it back into the service if using the strategy in this branch.